spring-cloud-stream-schema/src/main/java/org/springframework/cloud/stream/schema/client/config/SchemaRegistryClientProperties.java

```diff
@@ -28,6 +29,8 @@
 
 	private boolean enabled = true;
 
+	private boolean caching = true;
```

Comment:
I don't think it is, I can't really remember why we have it, I'll leave this one out, fixing all other comments on a new commit.